### PR TITLE
[DOCS] Guide for Viewing Results (show) and for Exporting them (json & html)

### PR DESCRIPTION
**Pull request by shir22**
#### Reference Issues/PRs

Implements #1709 

PDF's of new guides:
[showing_results.pdf](https://github.com/deepchecks/deepchecks/files/9175993/showing_results.pdf)
[export_save_results.pdf](https://github.com/deepchecks/deepchecks/files/9175992/export_save_results.pdf)

### CODE REVIEW

File: docs/source/user-guide/general/export_save_results.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
+=================================
+Exporting Deepchecks' Results
+=================================
+
+This guide will help you understand how to export deepchecks results, 
+which enables saving them for later viewing, sharing the analysis results as a report,
+sending to different tools, or for further processing steps 
+(e.g. in ci/cd for deciding whether a new model is good enough for deployment).
+
+We will will discuss the two main export options, that are relevant for saving Check or Suite results:
```
Comment (ItayGabbay) [inline docs/source/user-guide/general/export_save_results.rst:10]:
```suggestion
We will discuss the two main export options, that are relevant for saving Check or Suite results:
```

File: docs/source/user-guide/general/export_save_results.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  later be used in other Python scopes, and enables processing the checks' output with code.
+
+For sending the results to other tools (e.g. experiment trackers),
+you can either use one of the above send the desires results or report, or check out
+our :ref:`integrations examples section <user_guide__integrations>` in the user guide.
+
+
+.. _export_save_results__save_result_to_html:
+
+Save Result to an HTML Report: ``save_as_html()``
```
Comment (ItayGabbay) [inline docs/source/user-guide/general/export_save_results.rst:23]: Maybe link it with a :func:?

File: docs/source/user-guide/general/export_save_results.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+The suite output can also be written into a file buffers. This can be done by
+setting the file argument with a ``StringIO`` or ``BytesIO`` buffer object.
+
+
+.. code-block:: python
+
+    import io
+
+    html_out = io.StringIO()
+    suite_result.save_as_html(file=html_out)
```
Comment (ItayGabbay) [inline docs/source/user-guide/general/export_save_results.rst:51]: I don't think this is necessary

**Review by ItayGabbay — APPROVED** [review]

**Comment by ItayGabbay** [comment]
+ I think that the show_results guide should be in a more central location in the docs

**Comment by shir22** [comment]
About where it should be located - do you have a specific suggestion?
We did plan on linking to it from all of the quickstarts.

File: docs/source/user-guide/general/export_save_results.rst (hunk 3 above)
Comment (shir22) [reply docs/source/user-guide/general/export_save_results.rst:51]: agree :-) deleted... (origin was in the original guide)

File: docs/source/user-guide/general/export_save_results.rst (hunk 2 above)
Comment (shir22) [reply docs/source/user-guide/general/export_save_results.rst:23]: :meth:
added

**Comment by shir22** [comment]
closes #1709

**Pull request merged**